src/fortran_routing/Reservoir_Binding/Reservoirs/makefile

```diff
@@ -1,9 +1,9 @@
 NETCDFINC := $(shell nc-config --fflags)
 NETCDFLIB := $(shell nc-config --flibs)
-COMPILER90 = $(F90)
-F90FLAGS := -g -w -c -ffree-form -ffree-line-length-none -fconvert=big-endian -frecord-marker=4 $(NETCDFINC)
-F90LFLAGS := $(NETCDFLIB) -static-libgfortran -static-libgcc -nodefaultlibs -lgfortran -lgcc -fPIC
```

Comment:
regarding the linking here, do we need to use both `-static-libgfortran` and `-lgfortran`? Likewise for `-static-libgcc` and `-lgcc`...?

Reply:
Would have to defer to @hellkite500